docs/cue-lists/theatre-programming.md

```diff
@@ -9,9 +9,11 @@ more familiar with using a keypad/syntax style interface, rather than the Titan
 touchscreen. But fear not because you can program a theatre show on
 Titan using the keypad and this is how you do it.
 
-Labelled buttons on the console are shown like this: \<Enter\>. The
-\<Cue\> button is labelled \<Connect\> on some consoles. The \<@\>
-button is the one next to the numeric keypad.
+- Labelled buttons on the console are shown like this: \<Enter\>. 
```

Comment:
```suggestion
- Labelled buttons on the console are shown like this: \<Enter\>
```

Reply:
Well this is a totally minor thing and I don't really mind either way but I would have thought this should have a full stop since all the other bullet points have a full stop?